src/SimMatrix.cpp

```diff
@@ -309,3 +309,35 @@ SimMatrix SimMatrix::DoSimStepReturnMatrix(const MatrixSetup& setup)
 	*this = locSimMatrix;
 	return locSimMatrix;
 }
+
+// only for cells not from edges (with margins)
+void SimMatrix::MyLifeNeighbors(SimMatrix& sMatrix)
```

Comment:
Please do not use "my" - it sounds childish.

Reply:
https://www.youtube.com/watch?v=t1TcDHrkQYg xD